src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+---
+title: JavaScript Eventing Deep Dive
+subhead:
+  '`preventDefault` and `stopPropagation`: when to use which and what exactly each method does.'
+authors:
+  - stephenstchur
+  - thomassteiner
+Description: |
+  preventDefault and stopPropagationEvent handling in JavaScript is usually fairly
+  straightforward. Occasionally though, people get a little tripped up between
+  preventDefault and stopPropagation, when to use which one and what exactly each
+  method does. This article explains both with the goal of ending your confusion
+  once and for all.
+date: 2021-09-02
+# updated: 2021-08-26
+tags:
+  - blog
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/tPQ30G1QPF3ETAVf1Q40.jpg
+alt: Binders in many colors.
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/-6682215947110973439
+---
+
+`preventDefault` and `stopPropagationEvent` handling in JavaScript is usually fairly
+straightforward. Occasionally though, people get a little tripped up between `preventDefault` and
+`stopPropagation`, when to use which one and what exactly each method does. This article explains
+both with the goal of ending your confusion once and for all.
+
+Before even beginning a discussion of `preventDefault` or `stopPropagation` it's important to
+briefly touch on the two kinds of event handling possible in JavaScript (in all modern browsers that
+is—IE prior to version 9 did not support event capturing).
+
+### Eventing Styles (Capturing and Bubbling)
+
+All modern browsers support event capturing, but it is very rarely used by developers.
+Interestingly, it was the only form of eventing that Netscape originally supported. Netscape's
+biggest rival, Microsoft Internet Explorer, did not support event capturing at all, but rather, only
+supported another style of eventing called event bubbling. When the W3C was formed, they found merit
+in both styles of eventing and declared that browsers should support both, via a parameter to the
+`addEventListener` method. That parameter is the third parameter, after the event type and the event
+delegate, as follows:
+
+```js
+someElement.addEventListener('click', myClickHandler, useCapture);
+```
+
+It's that third parameter, `useCapture`, which determines if your event listener will listen in the
+capturing or the bubbling phase. Possible values are `true | false`. The default value is `false`
+meaning that event bubbling will be used most of the time.
+
+{% Aside %} The above syntax for `addEventListener` is the legacy syntax still commonly in use and
```

Comment:
I've reworked the copy so that the options object is prominent and the aside mentions where you can learn more about all of addEventListener's options (including its legacy sytax)